### DISCUSSION TITLE
Upload?

### DISCUSSION THREAD

**Discussion by lordvic1001** [crocodilestick/Calibre-Web-Automated#63] [Q&A] [answered]
Is there a button i can click to upload .EPUBs from my computer into Calibre Web Automated?

**Comment by iroQuai** [comment, marked as answer]
Yes, but you have to activate uploads in Admin tab >> Edit Basic Config >> Feature Configuration >> Check the Enable Uploads button

O and also on a per user basis you have to enable uploads in profile settings, if I remember correctly

**Reply by Tooelite** [reply]
What do you think about enabling upload by default? So this come more as a "out of the box" feature?

To fill up the database it is essential to enable uploading, so why not make it default? :-)

**Comment by lordvic1001** [comment]
I think that's a great idea.

Sent with [Proton Mail](https://proton.me/mail/home) secure email.
